test/development/app-dir/ssr-only-error/ssr-only-error.test.ts

```diff
@@ -40,4 +41,14 @@ describe('ssr-only-error', () => {
      }
     `)
   })
+
+  it('should not handle internal nextjs errors that will be handled by error boundaries', async () => {
+    const browser = await next.browser('/notfound')
```

Comment:
We should also check that the console is clear and no page errors are emitted:

```suggestion
    const browser = await next.browser('/notfound', { pushErrorAsConsoleLog: true })
```